src/to_egraph_serialized.rs

```diff
@@ -0,0 +1,54 @@
+use egraph_serialize::{ClassId, NodeId};
+use indexmap::IndexMap;
+
+use crate::ExtractionResult;
+
+pub fn get_term(
```

Comment:
writing documentation for this function would be appreciated! Looks like it prunes the e-graph based on the choices made by the extractor